### PR TITLE
Initial Hungarian translation

### PR DESCRIPTION
**Pull request by Cruel**
From smileyhead on GBATemp. Needs review.

### CODE REVIEW

File: res/romfs/lang/hungarian.lang [hunk 1]
```diff
@@ -0,0 +1,395 @@
+# src/States/SyncState.cpp
+##########################
+Waiting for internet connection... %.0fs
+Várakozás az internetkapcsolatra... %.0fs
```
Comment (smileyhead) [inline res/romfs/lang/hungarian.lang]: +Várakozás az internetkapcsolatra... %.0fmp

**Comment by smileyhead** [comment]
I've fixed a typo with a comment. Sorry for the inconvenience.

**Comment by Cruel** [comment]
Thanks. Also, what's the translation for the missing "Installed" text? I'll add that.

**Comment by smileyhead** [comment]
Telepítve

File: res/romfs/lang/hungarian.lang [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+Seed telepítése...
+
+Installing TMD...
+TMD telepítése...
+
+Importing content...
+Tartalom importálása...
+
+Installing %d/%d... %.1f%% (%.0f KB/s) %dm %02ds
+Telepítés %d/%d... %.1f%% (%.0f KB/s) %dp %02dmp
```
Comment (Cruel) [inline res/romfs/lang/hungarian.lang]: If you changed "s" to "mp" elsewhere, then should it be "KB/mp"?

**Comment by smileyhead** [comment]
Lol you're right, I forgot to change s (second(s)) to mp (másodperc) on a lot of places. I'll do it tomorrow. It's 11 PM already.

File: res/romfs/lang/hungarian.lang (hunk 2 above)
Comment (smileyhead) [inline res/romfs/lang/hungarian.lang]: +Telepítés %d/%d... %.1f%% (%.0f KB/mp) %dp %02dmp

File: res/romfs/lang/hungarian.lang [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+Filter-beállítások elmentve
+
+Cleared saved filter settings
+FIlter-beállítások kitörölve
+
+Select All
+Mind
+
+Select None
+Egyet se
```
Comment (smileyhead) [inline res/romfs/lang/hungarian.lang]: +Egy sem

**Comment by smileyhead** [comment]
Okay, I've made some fixes (sorry for being late; my memory is a bitch), it's ready for release. :)

**Comment by smileyhead** [comment]
By the way, something I just noticed: the genres and the platforms aren't in the language file, and I can't translate them. :-/

**Comment by Cruel** [comment]
Yeah, they're fetched from eShop, so it doesn't have translations for non-standard languages. Oh well...

**Comment by smileyhead** [comment]
Australia - Ausztrália
China - Kína
Korea - Korea
Taiwan - Taiwan

**Comment by Cruel** [comment]
Thanks!

**Comment by smileyhead** [comment]
You're welcome. :-)

**Pull request merged**